apps/editing-toolkit/editing-toolkit-plugin/help-center/class-wp-rest-help-center-fetch-post.php

```diff
@@ -34,6 +34,53 @@ public function register_rest_route() {
 				'permission_callback' => array( $this, 'permission_callback' ),
 			)
 		);
+		register_rest_route(
+			$this->namespace,
+			'/articles',
+			array(
+				'methods'             => \WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_articles_fetch_results' ),
+				'permission_callback' => array( $this, 'permission_callback' ),
+				'args'                => array(
+					'blog_id'  => array(
+						'type'     => 'number',
+						'required' => true,
+					),
+					'post_ids' => array(
+						'type'     => 'array',
+						'required' => true,
+						'items'    => array(
+							'type' => 'string',
+						),
+					),
+				),
+			)
+		);
+	}
+
+	/**
+	 * Should fetch articles
+	 *
+	 * @param \WP_REST_Request $request    The request sent to the API.
+	 *
+	 * @return WP_REST_Response
+	 */
+	public function get_articles_fetch_results( \WP_REST_Request $request ) {
```

Comment:
Let's change the function name and comment here.

Maybe `get_articles` or `get_blog_post_articles`?

Reply:
`get_blog_post_articles` reads great to me 👍